OpenRA.Game/Traits/Health.cs

```diff
@@ -188,6 +188,15 @@ public static DamageState GetDamageState(this Actor self)
 			return (health == null) ? DamageState.Undamaged : health.DamageState;
 		}
 
+		public static bool IsDamaged(this Actor self)
```

Comment:
There is no reason for me to call the float math if I don't need to, but I can if you guys find it necessary.